ros2_ws/src/rcdt_gz_worlds/launch/table.launch.py

```diff
@@ -0,0 +1,49 @@
+# SPDX-FileCopyrightText: Alliander N. V.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import List
+
+from launch import LaunchContext, LaunchDescription
+from launch.actions import (
+    IncludeLaunchDescription,
+    OpaqueFunction,
+)
+from launch_ros.actions import Node
+from rcdt_utilities.launch_utils import LaunchArgument, get_file_path
+
+load_gazebo_ui_arg = LaunchArgument("load_gazebo_ui", True, [True, False])
+world_arg = LaunchArgument("world", "table_brick.sdf")
```

Comment:
De naam van deze launchfile suggereert dat er altijd een table-world gelanceerd wordt. Dit als launchargument is dan onverwacht